src/data/data1d/filter1d/autoPhaseCorrection.js

```diff
@@ -12,17 +12,17 @@ export const name = 'Automatic phase correction';
  */
 
 const defaultOptions = {
-  magnitudeMode: false,
-  minRegSize: 30,
-  maxDistanceToJoin: 256,
+  minRegSize: 10,
+  maxDistanceToJoin: 128,
+  magnitudeMode: true,
+  factorNoise: 5,
 };
 
 export function apply(datum1D, options = {}) {
   if (!isApplicable(datum1D)) {
     throw new Error('phaseCorrection not applicable on this data');
   }
   options = Object.assign({}, defaultOptions, options);
```

Comment:
Please use the new syntax
`options = {...defaultOptions, ...options}`

Reply:
you can also directly pass the spread to the function call: `reimAutoPhaseCorrection(datum1D.data, {...defaultOptions, ...options})`